Firestore/Source/Public/FirebaseFirestore/FIRVectorValue.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Represent a vector type in Firestore documents.
+ * Create an instance with `@link `FieldValue.vector(...)`.
+ */
+NS_SWIFT_NAME(VectorValue)
+@interface FIRVectorValue : NSObject
+
+/** Returns a copy of the raw number array that represents the vector. */
+@property(readonly) NSArray<NSNumber *> *array NS_REFINED_FOR_SWIFT;
+
+/** :nodoc: */
+- (instancetype)init NS_UNAVAILABLE;
+
+/**
+ * Creates a `VectorValue` constructed with a copy of the given array of NSNumbrers.
```

Comment:
If you're changing Creates -> Represents per egilmore comment on the js sdk version, apply same changes here AND in the Android version.

Otherwise LGTM.

Reply:
Thanks. Updated.